qmxgraph/page/graphs.js

```diff
@@ -431,6 +431,16 @@ graphs.createGraph = function createGraph (container, options, styles) {
     };
     graph.addListener(mxEvent.REFRESH, onRefresh);
 
+    // * Adds mouse wheel handling for zoom
+    mxEvent.addMouseWheelListener(function(evt, up) {
```

Comment:
What's `up` means?

Reply:
Is the direction my mouse wheel must move to move the pages' scrollbar up. I think that is the default configuration.

Reply:
I see thanks. Please document that.

Reply:
Added a comment explaining `up`.